### PR TITLE
Update docs to reference multiple terminals and other improvements

### PR DESCRIPTION
**Pull request by santiagocardenas**
*Description of changes:*
Updates to kmstool doc steps to include:
* references to different terminal sessions
* note on instance profile associated with instance
* note on log out and back in for related Nitro Enclaves CLI user permissions to take effect
* updates to memory allocation to account for EIF requirements

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by petreeftime — DISMISSED** [review]

File: docs/kmstool.md [hunk 1]
```diff
@@ -244,7 +258,7 @@ for securing and managing production applications.
 The instructions for running the sample are given for both Linux and Windows.
 Windows instructions are preceded with `PowerShell`
 
-Start the enclave:
+In a second (new) terminal session, start the enclave in debug mode and connect:
 ```sh
 nitro-cli run-enclave --eif-path kmstool.eif --memory 512 --cpu-count 2 --debug-mode
```
Comment (andraprs) [inline docs/kmstool.md:263]: Can you please also update the memory value here to 1024 instead of 512.

Comment (andraprs) [reply docs/kmstool.md:263]: Same below, for the PowerShell section.

Comment (santiagocardenas) [reply docs/kmstool.md]: Updated!

**Comment by santiagocardenas** [comment]
I've also expanded on the note here: https://github.com/aws/aws-nitro-enclaves-sdk-c/pull/56/files#diff-e080d68ae455b8ee7996748ffdce3ef1e1bc64117c0503e214fa27c8a7078587L358-R372

**Review by andraprs — APPROVED** [review]

File: docs/kmstool.md (hunk 1 above)
Comment (andraprs) [reply docs/kmstool.md]: Thanks.

**Comment by santiagocardenas** [comment]
@andraprs @petreeftime apologies for the last commit. I realized the sentence linking to docs wasn't structured correctly.

**Review by andraprs — APPROVED** [review]

**Review by popegeo — APPROVED** [review]

**Pull request merged**